R/MetaDataDiagnostics.R

```diff
@@ -164,4 +166,7 @@ getVocabularyVersion <- function(connection, vocabularyDatabaseSchema) {
     dplyr::rename(vocabularyVersion = .data$vocabularyVersion) %>%
     dplyr::pull(.data$vocabularyVersion) %>%
     unique()
+
+  # Edge case where a CDM has more than a single entry
+  paste(vocabularyVersion, collapse = ";")
```

Comment:
I agree, its a violation of the con straints but this is what the VA are doing so we'll need to support it. This won't have a major impact in any of the cohort diagnostics but it will tell us that users have a funky CDM.

Reply:
The output of "SELECT * from vocabulary" should not violate the conformance standards expected by OMOP. Can we confirm if this is TRUE i.e. new fields are being added to the vocabulary table that is a primary key violation?

again - i am fine with cdm_source table, but this vocabulary table is making me uneasy.

Reply:
After discussion with @azimov i think it is fine